### PR TITLE
ImageBitmapLoader: Added documentation

### PR DESCRIPTION
**Pull request by Mugen87**
Follow-up of: #12695

### CODE REVIEW

File: docs/api/loaders/ImageBitmapLoader.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		<h3>[method:null load]( [page:String url], [page:Function onLoad], [page:Function onProgress], [page:Function onError] )</h3>
+		<div>
+		[page:String url] — the path or URL to the file. This can also be a
+			[link:https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs Data URI].<br />
+		[page:Function onLoad] — Will be called when load completes. The argument will be the loaded [page:Image image].<br />
+		[page:Function onProgress] — Will be called while load progresses. The argument will be the progress event.<br />
+		[page:Function onError] — Will be called when load errors.<br />
+		</div>
+		<div>
+		Begin loading from url and return the [page:Image image] object that will contain the data.
```
Comment (donmccurdy) [inline docs/api/loaders/ImageBitmapLoader.html]: Return type is ImageBitmap here.

File: docs/api/loaders/ImageBitmapLoader.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+		<base href="../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		<h1>[name]</h1>
+
+		<div class="desc">
+			A loader for loading an [page:Image] as an [link:https://developer.mozilla.org/de/docs/Web/API/ImageBitmap ImageBitmap].
```
Comment (donmccurdy) [inline docs/api/loaders/ImageBitmapLoader.html]: Perhaps add a note about why someone might want to use this, e.g.:

> An `ImageBitmap` provides an asynchronous and resource efficient pathway to prepare textures for rendering in WebGL.

File: docs/api/loaders/ImageBitmapLoader.html (hunk 1 above)
Comment (Mugen87) [reply docs/api/loaders/ImageBitmapLoader.html]: 👍

File: docs/api/loaders/ImageBitmapLoader.html (hunk 2 above)
Comment (Mugen87) [reply docs/api/loaders/ImageBitmapLoader.html]: @WestLangley I remember we said some time ago that the documentation should not focus on tutorial stuff. Therefore i struggle a little but with adding an explanation of `ImageBitmap`. What do you think?

**Review by donmccurdy — APPROVED** [review]

File: docs/api/loaders/ImageBitmapLoader.html (hunk 2 above)
Comment (mrdoob) [reply docs/api/loaders/ImageBitmapLoader.html]: I think the short explanation Don suggests is good though. One can go to MDN after that is needed.

This is pretty much (sadly) Chrome only at the moment anyway 🙁

Comment (donmccurdy) [reply docs/api/loaders/ImageBitmapLoader.html]: Ok by me with or without the explanation. ImageBitmap may also be too new to justify writing best practices for its use — still not working in Firefox or Safari Tech Preview for me here, despite partial implementation. Down the road I see it as a recommended way to load images into an already-running scene, but not necessarily better for up-front initialization.

Comment (WestLangley) [reply docs/api/loaders/ImageBitmapLoader.html]: @Mugen87 I agree with @mrdoob here. I know this is not a black-and-white issue.

Comment (Mugen87) [reply docs/api/loaders/ImageBitmapLoader.html]: Ok, i've added the suggestion of @donmccurdy 👍

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**